contracts/package.json

```diff
@@ -7,9 +7,7 @@
   "author": "James Zaki",
   "license": "MIT",
   "dependencies": {
-    "@openzeppelin/contracts": "^4.3.2",
-    "bls-wallet-clients": "0.3.0-9bdca2f",
-    "bls-wallet-signer": "^0.8.0-rc.1"
```

Comment:
If you want to keep the imports the same as elsewhere, you should be able to use a local path to install here. https://docs.npmjs.com/cli/v6/configuring-npm/package-json#local-paths